bin/disco_snapshot.py

```diff
@@ -60,6 +60,8 @@ def get_parser():
     parser_take_group.add_argument('--hostclass', dest='hostclasses', default=[], action='append', type=str)
     parser_take_group.add_argument('--ami', dest='amis', default=[], action='append', type=str)
     parser_take_group.add_argument('--volume-id', dest='volume_id', type=str)
+    parser_take_group.add_argument('--disk-usage', dest='disk_usage', required=False, type=str,
```

Comment:
It might actually be a good idea to make this into a generic `tags` field. Then `take_snapshot.sh` can pass the disk usage with something like `--tag disk_usage:30MB`. We could then use it for tagging the snapshot with other things too.